src/entity-configuration/definitions/fields-defs/model.tsx

```diff
@@ -115,12 +119,93 @@ export const FieldsDefinition: Partial<FieldsDefinitionRegistry<EntityCoreObject
   [EntityCoreFields.SynaptomeUsedMEModelName]: {
     className: 'text-center',
     title: 'ME-model',
-    filter: null,
-    render: (r) => renderEmptyOrValue((r as ISingleNeuronSynaptome).me_model.name),
+    filter: CoreFieldFilterTypeEnum.CheckList,
```

Comment:
i dont think we have a filter for `me-model` right now in listing page, this should be (for the moment)
```suggestion
    filter: nil 
    isFilterable: false
    isDisplayable: false,
```